tools/testing/target_determination/determinator.py

```diff
@@ -27,3 +27,10 @@ def get_test_prioritizations(tests: List[str]) -> AggregatedHeuristics:
             new_rankings.print_info()
 
     return aggregated_results
+
+
+def get_experimental_ratings(tests: List[str]) -> Dict[str, Dict[str, float]]:
```

Comment:
Suggestion: Seems like it gets the ratings for all tests

Though if you accept renaming get_test_ratings as per the other suggestion, this would be renamed accordingly as well
 
```suggestion
def get_ratings(tests: List[str]) -> Dict[str, Dict[str, float]]:
```